src/main/scala/com/mwz/sonar/scala/scapegoat/ScapegoatRulesRepository.scala

```diff
@@ -0,0 +1,50 @@
+package com.mwz.sonar.scala
```

Comment:
Oh of course, I'm sorry.

Reply:
PS: I just realize that the scapegoat inspections autogenerated file doesn't have the copyright header either. Would you like it to have it?, or since it is outside of the repo there is no problem.
(Note: I think I already asked this, but I could not remember, and I tried to search for it in previous comments but I could not find nothing).

Reply:
It's probably not necessary since the generated file is not versioned.

Reply:
ok